### ISSUE TITLE
Add a "save demo" button on the Round Statistics screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/hedgewars#264]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?

Would be nice and way easier to save a demo if you add a button below the 
health graph. because if we want to save our last round, we must close our room 
with all friends in it, and then go to the main screen, etc, etc. 


What version of the product are you using? On what operating system?
0.9.15 (mac app store) OS X Lion

Please provide any additional information below.

i added a picture :)
```

-----
Original issue reported on code.google.com by `iiigerar...@gmail.com` on 23 Aug 2011 at 6:10

Attachments:
* [hedgeround.jpg](https://storage.googleapis.com/google-code-attachments/hedgewars/issue-264/comment-0/hedgeround.jpg)

**Comment by GoogleCodeExporter** [comment]
```
... All games are saved automatically.

If you want to not overwrite the last game with a new one, check off the 
timestamp option in the Config.

```

-----
Original comment by `kyberneticist@gmail.com` on 24 Aug 2011 at 4:11
* Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
How is this an invalid suggestion? It's quite understandable to not want to 
save ALL games automatically (especially when demos are named with just a 
timestamp), but only a select few.
```

-----
Original comment by `gramm...@gmail.com` on 25 Aug 2011 at 11:43

**Comment by GoogleCodeExporter** [comment]
```
how about adding the save button and prompting the user a place where to save 
the demo?
```

-----
Original comment by `vittorio...@gmail.com` on 25 Aug 2011 at 12:00

**Comment by GoogleCodeExporter** [comment]
```
"if we want to save our last round, we must close our room with all friends in 
it, and then go to the main screen"

I interpreted this drastic action to mean they were completely unaware that 
there was an option to automatically save each game independently.

Not that they insisted on closing the room just to give the game a distinctive 
name...

Clearly you see it differently, but that's why I closed it invalid, as I felt 
they were unaware of existing functionality.
I still see it that way, but I guess we won't know until they answer.

As for koda, I find it amusing that you are suggesting everyone  in the room 
get prompted.  After every game.  I thought you weren't a fan of even a single 
prompt on login. :)

```

-----
Original comment by `kyberneticist@gmail.com` on 25 Aug 2011 at 1:35

**Comment by GoogleCodeExporter** [comment]
```
Issue 267 has been merged into this issue.
```

-----
Original comment by `kyberneticist@gmail.com` on 25 Aug 2011 at 4:06

**Comment by GoogleCodeExporter** [comment]
```
Regardless of original intent of this bug, claymore's filing of a new one 
suggests he wants this feature soooo, rather than clutter up bug db more...
```

-----
Original comment by `kyberneticist@gmail.com` on 25 Aug 2011 at 4:17
* Changed state: **New**
* Added labels: **Type-Enhancement**, **Priority-Low**
* Removed labels: **Type-Defect**, **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
sorry for my bad explanation but yeah, i mean, i don't want to save every game 
i play, i just want an option to let me save a demo of the game that i just 
played.

Let's say i did something incredible, the round ends, i just click the save 
demo button, done :)
```

-----
Original comment by `iiigerar...@gmail.com` on 26 Aug 2011 at 4:43

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision d710db47a1ef.
```

-----
Original comment by `unC0Rr` on 2 Sep 2011 at 8:07
* Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
save game button is present on game statistics
```

-----
Original comment by `jerryma1...@gmail.com` on 21 Dec 2011 at 1:04
* Changed state: **Verified**
